source/computer/linux101.rst

```diff
@@ -1,2 +1,54 @@
 Linux 入门
 ==========
+
+:本节贡献者:
+:最近更新日期:
+:预计花费时间:
+
+----
+
+命令行
+-------
+
+命令行与图形界面
+^^^^^^^^^^^^^^^^
+
+终端
+^^^^^
+
+文件管理
+---------
+
+熟悉 Linux 文件系统
+^^^^^^^^^^^^^^^^^^^
+
+操作文件与目录
+^^^^^^^^^^^^^^
+
+文件路径
+^^^^^^^^^
+
+文件权限
+^^^^^^^^
+
+Shell 基础
+----------
+
+重定向
+^^^^^^
+
+管道
+^^^^
+
+通配符
+^^^^^^
+
+正则表达式
+^^^^^^^^^^
+
+环境变量
+--------
+
```

Comment:
Don't environmental variables belong to Shell?

Reply:
Maybe, but I feel that environmental variables are more important so a new subsection.

Reply:
- `PATH`
- `~/.bashrc`
- `/etc/hosts`

right?

Reply:
Yes for PATH and bashrc. 

I don't understand why you add /etc/hosts here.

Reply:
We may use it to rename our computer, right?

Reply:
If we use a server, we may add server IP and name in this file/

Reply:
It makes no difference for users, so I don't think it belongs to 101.